getting-started/index.rst

```diff
@@ -1,31 +1,126 @@
 
-Getting Started
-###############
+Getting Started with OPEA
+#########################
 
-1. Try with AI Catalog
+The OPEA ecosystem consists of several complete :ref:`GenAI Examples <GenAIExamples>`
+put together using building-block like :ref:`GenAI microservices <GenAIMicroservices>`.
+You can run these examples as provided, and then customize them to your own
+implementation's needs.or You can also build your own solutions from scratch
+using these microservices.
 
-#. ChatQnA summary
+See these and other Powered by OPEA solutions in action on the
+`AI Explorer`_ site, hosted by Intel:
 
-   a. Default LLM used, additional LLM supported
+.. image:: images/AI-Explorer.png
+   :width: 600px
+   :align: center
+   :class: drop-shadow
+   :target: https://aiexplorer.intel.com/explore
 
-      1. How modify LLMs
+.. _AI Explorer: https://aiexplorer.intel.com/explore
 
-   #. Knowledge base
+-----
 
-      1. Default RAG knowledge base
-      #. BKMs for knowledge base customization
-      #. How to use your own knowledge base
+Get Started
+***********
 
-#. Deployment options
+Prerequisites
+=============
 
-   a. Cloud
-   #. AI enabled PC
+* Python
+* Docker
+* HuggingFace API Token
+* Prometheus and Grafana (optional, needed for evaluation purposes)
 
-      1. Supported platforms
-      #. Deployment
-      #. ChatQnA customization.
+1. **Determine where you will run your solution.**
 
-   #. Complete Deployment Options (links to Deployment section)
+   a. Basic deployment options suitable for proof of concept includes single architectures such as:
+
+      * An AI enable PC.
+      * A server with Intel Xeon Processors.
+
+   b. Production ready environments include:
+
+      * Onprem or cloud based environments (including environments with NVIDIA GPUs).
+      * Intel Gaudi Platforms.
+      * Multinode deployments using Kubernetes.
+
+2. **Deploy a simple chat bot application using the OPEA ChatQnA example into your
+   environment.**
+
+3. **Customize the chat bot with your own data (RAG model)**.
+
+4. **Evaluate the performance and accuracy of your chat bot project.**
+
+   * A four-step assessment for grading generative AI systems around
+     performance, features, trustworthiness, and enterprise-grade readiness.
+
+Next Steps
+==========
+
+* Explore additional :ref:`GenAI Examples <GenAIExamples>`
+* Build your own solution using :ref:`GenAI microservices <GenAIMicroservices>`
+* Get involved and contribute to the :ref:`OPEA open source community <OPEA_community>`
+
+Like What You See?
+******************
+
+Do you have ideas and skills to build out genAI components, microservices, and
+solutions? Want to be part of these  early stages of this evolving technology?
+
+Get Involved
+============
+
+Register for our mailing list:
+  * General: https://lists.lfaidata.foundation/g/OPEA-announce
+  * Technical Discussions: https://lists.lfaidata.foundation/g/OPEA-technical-discuss
+
+Subscribe to working group mailing lists
```

Comment:
s/Subscribe to working group mailing lists/Subscribe to one or more of the working group mailing lists based on your interest: